<!--DOCTYPE html-->
<launch>

  <node pkg="mpu6050_mappleboard_driver" type="mpu6050_mappleboard_driver" name="mpu6050_mappleboard_driver" output="screen">
  	<param name="serial_name" value="/dev/ttyACM0"/>
  	<param name="topic_name" value="/imu_mpu6050"/>
  	<param name="frame_id" value="right_foot"/>
  </node>

</launch>
